src/components/combine.rs

```diff
@@ -12,6 +12,7 @@ use crate::{
     scripts::{escrow_scripts, escrow_spend_info},
     sign::combine_signatures,
     util::{days_to_blocks, hours_to_blocks, parse_escrow_type, parse_npub},
+    validation::*,
```

Comment:
no globbing import the stuff explicitly, this is not Python...
We only glob preludes like `dioxus::prelude::*` or in tests with `use super::*`